lib/docs.js

```diff
@@ -0,0 +1,19 @@
+// Create object for providing easy access to documentation
+
+function Docs() {
+  var fs = require('fs'),
+      path = require('path'),
+      rules = fs.readdirSync(path.resolve('node_modules/eslint/docs/rules')),
```

Comment:
@wfleming Tests sound good!

Sent from my iPhone

> On Oct 17, 2015, at 11:16 AM, Will Fleming <notifications@github.com> wrote:
> 
> In lib/docs.js:
> 
> > @@ -0,0 +1,19 @@
> > +// Create object for providing easy access to documentation
> > +
> > +function Docs() {
> > +  var fs = require('fs'),
> > +      path = require('path'),
> > +      rules = fs.readdirSync(path.resolve('node_modules/eslint/docs/rules')),
> Can we determine & resolve this path relative to __dirname? I think this would be less fragile (and easier to test).
> 
> Also, I know I'm a broken record, but can we write tests? 
> 
> —
> Reply to this email directly or view it on GitHub.
>